Versions page

sbkobaidze opened the pull request:
Displays most recent commits from OSFD-Frontend's commit page using Github Octokit API. By saving commit data into the browser's localStorage, React-Toastify notifies active user when new commits are made

nsafarova committed:
media-page-update

nsafarova committed:
media-page-update

BrandonC123 committed:
Fix needing to double click mint button for when wallet is already connected.

BrandonC123 committed:
Fix double click when wallet is not yet connected and update button if wallet disconnects.

sbkobaidze committed:
created versions page

sbkobaidze committed:
some changes

sbkobaidze committed:
Added set interval

nsafarova committed:
update to package-lock.json

sbkobaidze committed:
missing dependency warning solved

sbkobaidze committed:
s

sbkobaidze committed:
edited auth key

YuWei-CH committed:
Update with Builder Sandbox Instruction

1. make "here" link as button and click to open a new tab.
2.Add border to input box

sbkobaidze committed:
small change

YuWei-CH committed:
Fix security risk Error

Error: Using target="_blank" without rel="noreferrer" (which implies rel="noopener") is a security risk in older browsers

BrandonC123 committed:
Merge branch 'main' of https://github.com/OSFD-Winter/OSFD-Frontend into double-click-mint-fix

BrandonC123 committed:
Fix CodeFactor error

tanerdurmaz commented:
automating this page is cool idea, I checked [deployment](https://osfd-frontend-git-fork-sbkobaidze-ve-b3c531-personaltestingosfd.vercel.app/version)  it doesnt work we get bad credentials error, alsolets fix conflicts on package-lock (dont manually do it, delete that file and node modules then use npm i)

tanerdurmaz committed:
Merge pull request #80 from OSFD-Winter/double-click-mint-fix

Double click mint fix

sbkobaidze committed:
re-installed node-modules & updated API Key

sbkobaidze committed:
test

sbkobaidze committed:
edited path-ignore path

sbkobaidze committed:
test

sbkobaidze committed:
added new file to ignore from secret scanner

sbkobaidze committed:
test

sbkobaidze committed:
DISABLE_SECRET_SCANNING

YuWei-CH committed:
Merge branch 'OSFD-Winter:main' into main

YuWei-CH committed:
Update sandbox.tsx

1. make all buttons without border and white_bgcolor
2. button the first "Here" and make it jump to new page after click
3. modify the layout of "Choose File" and "Upload" button
4. Add "Cancel" and "Create NFT Series" buttons.

Next to do:  Complete the functions of the above two buttons

YuWei-CH committed:
Fix error in CodeFactor Check

Fix:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.

tanerdurmaz committed:
Merge pull request #83 from YuWei-CH/main

Update Sanbox building instruction

nsafarova committed:
resolved problems

nsafarova committed:
fixed errors

tanerdurmaz committed:
Merge pull request #76 from OSFD-Winter/media-page-update

Media page update

sbkobaidze committed:
test

sbkobaidze committed:
added api key in env file

tanerdurmaz committed:
interns dao neo-brutalism

sbkobaidze committed:
test

sbkobaidze committed:
updated
